src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -15,13 +15,15 @@
 # limitations under the License.
 """PyTorch OpenAI GPT-2 model."""
 
+import math
 import os
 from dataclasses import dataclass
 from typing import Optional, Tuple
 
 import torch
 import torch.utils.checkpoint
 from torch import nn
+from torch.cuda.amp import autocast
```

Comment:
Is this supported by PyTorch 1.4.0 and higher? We try to keep backwards compatibility until PyTorch 1.4.0 I think -> we might need a 

```
if transformers.__version__ < ....
```

if PyTorch 1.4.0 doesn't support amp

Reply:
@LysandreJik

Reply:
Should now be resolved with an explicit version check!